explainer.md

```diff
@@ -55,28 +55,28 @@ Provide web apps with efficient access to built-in (software and hardware) media
 
 We build on top of [WHATWG Streams](https://streams.spec.whatwg.org/) (in particular [TransformStreams](https://streams.spec.whatwg.org/#ts-class)) and [MediaStreamTracks](https://www.w3.org/TR/mediacapture-streams/#dom-mediastreamtrack).
```

Comment:
I wouldn't say that the current design is built on MediaStreamTracks, just that we hope to integrate well with them.

Reply:
Done